packages/ra-ui-materialui/src/button/SaveButton.tsx

```diff
@@ -54,7 +54,6 @@ export const SaveButton = <RecordType extends RaRecord = any>(
         label = 'ra.action.save',
         onClick,
         mutationOptions,
-        saving,
```

Comment:
`saving` should be removed from the props type as well (I believe we can consider it's not a BC because it was not documented in the first place)